src/main/java/org/mule/extension/http/internal/request/HttpRequester.java

```diff
@@ -247,12 +247,19 @@ private void checkIfRemotelyClosed(Throwable exception, UriParameters uriParamet
     }
   }
 
-  private boolean shouldRetryRemotelyClosed(Throwable exception, int retryCount, String httpMethod) {
+  private boolean shouldRetryRemotelyClosed(Throwable exception, int retryCount, HttpRequest httpRequest) {
     boolean shouldRetry = exception instanceof IOException && containsIgnoreCase(exception.getMessage(), REMOTELY_CLOSED)
-        && supportsRetry(httpMethod) && retryCount > 0;
+        && supportsRetry(httpRequest.getMethod()) && retryCount > 0;
     if (shouldRetry) {
-      logger.warn("Sending HTTP message failed with `" + IOException.class.getCanonicalName() + ": " + REMOTELY_CLOSED
-          + "`. Request will be retried " + retryCount + " time(s) before failing.");
+      boolean entitySupportRetry = entitySupportRetry(httpRequest);
+      if (entitySupportRetry) {
+        logger.warn("Sending HTTP message failed with `" + IOException.class.getCanonicalName() + ": " + REMOTELY_CLOSED
```

Comment:
I would set this as debug as this was the previous normal behavior. To avoid performance degradation verify that debug is enabled.